agent/utils/host_watchdog.py

```
@@ -0,0 +1,468 @@
+
+    正常退出时由 Transceiver 的析构负责删除，但宿主失联后我们只能 os._exit()
+    （见 main.py 的说明），析构不会跑，所以在这里补一刀。
+    纯数字的 identifier 是 TCP 模式，没有文件。
+    """
+    if not socket_id or socket_id.isdigit():
+        return
+    try:
+        from pathlib import Path
+
+        # 路径与 AgentCommon/Transceiver.cpp 的 temp_directory() 保持一致
+        base = Path("C:/Temp") if sys.platform == "win32" else Path("/tmp")
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)